web/src/components/RelatedDestinations/RelatedDestinations.tsx

```diff
@@ -98,7 +98,7 @@ const RelatedDestinations: React.FC<RelatedDestinationsSectionProps> = ({
 
   return (
     <Flex align="center" spacing={2} mt={1}>
-      {destinations.map(getLogo)}
+      {destinations.length ? destinations.map(getLogo) : 'N/A'}
```

Comment:
```suggestion
      {destinations.length ? destinations.map(getLogo) : 'Not configured'}
```

I think it's a bit more explicit likee that, since N/A maps to "something is not available". Perhaps we need a different text color for that as well like the one we use in `GenericItemCard.Value` when something isn't available

Reply:
Updated to `Not Configured` with smaller `fontSize` to takes less space